draft-ietf-quic-transport.md

```diff
@@ -1156,6 +1156,31 @@ SHOULD ignore any such packets.
 
 Servers MUST drop incoming packets under all other circumstances.
 
+### Considerations for simple load balancers
+
+A server deployment could load balance among servers using simply source and
+destination IP addresses and ports. Changes to the client's IP address or port
+could result in packets being forwarded to the wrong server. Such a server
+deployment could use one of the following methods for connection continuity
+when a client's address changes.
+
+* Servers could use an out-of-band mechanism to forward packets to the correct
+server based on Connection ID.
+
+* If servers can use other dedicated server IP addresses or ports than
+the one that the client is initiating connections to, they could use the
+preferred_address transport parameter to request that clients move
+connections to these dedicated addresses. Note that clients could
+choose not to use the preferred address.
```

Comment:
```suggestion
* If servers can use a dedicated server IP address or port, other than the one
that the client is initially connects to, they could use the preferred_address
transport parameter to request that clients move connections to a dedicated
address. Note that clients could choose not to use the preferred address.
```

Reply:
Isn't the whole point of this section that the address/port initially used by the client _isn't_ dedicated to this particular server?  It seems like the entire "other than... connects to" phrase is unnecessary.